protocol/protocol.md

```diff
@@ -25,42 +25,41 @@ commands that are querying information.
 * Connect
     * Request
         * `0c` - Command.
-        * `00 0a` - ?? fixed.
-        * `eb 2b 76 14 0c 1b 07 15 35 04` - ?? varying with each connection.
-          Is `0a` an array length for this?
+        * `00 0a` - command length in number of bytes.
+        * `eb 2b 76` - random bytes that are only guaranteed to not be 5a or a5 (start and end values).
+        * `14 0c 1b 07 15 35 04` - year, month, day, day of week, hour, minute, second.
     * Response
         * `c9` - ?? varying.
-        * `01` - ?? varying 00 or 01.
-        * `09 54 72 69 6d 6c 69 67 68 74` - "Trimlight" (assuming 09 is a string length).
-          Controller name.
-        * `00 00` - ?? fixed.
-        * `02 58` - Pixel count? **TODO: Confirm this with a different system.*
+        * `00 09` - length of controller name.
```

Comment:
I don't think this is a two-byte length? The example I provided in the diffbase has 01 for first byte.

Reply:
Nope, it's two length in all my testing. I've never seen it represent anything else. And I've changed the name of my controller several times and the value has changed accordingly. The decompiled source code that I have been using to research also indicates the same thing.

Reply:
@msjarrett can you point me to where you have seen the first value of what I believe to be a two byte length indicator ever being `01` are so I can look into it more.

Reply:
msjarrett-trimlight1.pcapng; packet 134.

Reply:
After double checking the decompiled code and looking into more, it appears to be a random value that is not used that I can see. I have fixed the description to reflect that you are correct and I misremembered.